csun-metalab/nomi
# HOTFIX-11-7-2018-logoutBug

# Description
Hopefully fixed login bug and added max-width to student profile image.
  
# Testing
1. Login on steve; logout and login to mattew.fritz.
2. Ensure the state is cleared on login and properly show the correct faculty data.
3. Visit a student profile; resize window and make sure the student image doesn't grow past 250px.
